build-tests/api-extractor-test-01/src/TypeLiteral.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
I think the `api-extractor-scenarios/apiItemKinds` test project could be a better fit for this test case. In the past, I've used the `api-extractor-scenarios` project to hold different test cases that don't require a standalone project set up. We could create a new file called `typeAliases.ts` there, and add some supported vs unsupported type alias examples. I think the `api-extractor-test-01` test project tests something very specific (despite its ambiguous name). WDYT?